File: RepoTransBench/target_projects/C/Python/gurugio_book_cprogramming/tests/original/test_error_handle.py
import unittest
from src.gurugio_book.error_handle import error_printer, error_return

class TestErrorHandle(unittest.TestCase):
    """Tests for the error handling functions translated from test_error_handle.c"""
    
    def test_error_printer(self):
        """Test the error_printer function"""
        # coverage: normal use
        self.assertEqual(error_printer("hello world"), 0)
        # coverage: NULL (error branch)
        self.assertEqual(error_printer(None), -1)
    
    def test_error_return(self):
        """Test the error_return function"""
        # coverage: normal (error code)
        self.assertEqual(error_return(5), 5)
        # coverage: no print branch
        self.assertEqual(error_return(0), 0)

if __name__ == '__main__':
    unittest.main()
    print("test_error_handle: OK")

File: RepoTransBench/target_projects/C/Rust/gurugio_book_cprogramming/tests/original/test_error_handle.rs
use gurugio::error_printer;
use gurugio::error_return;

#[test]
fn test_error_printer() {
    // coverage: normal use
    assert_eq!(error_printer(Some("hello world")), 0);
    // coverage: NULL (error branch)
    assert_eq!(error_printer(None), -1);
}

#[test]
fn test_error_return() {
    // coverage: normal (error code)
    assert_eq!(error_return(5), 5);
    // coverage: no print branch
    assert_eq!(error_return(0), 0);
}